<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLogin</class>
 <widget class="QWidget" name="FormLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_log" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>1081</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_log{

}</string>
   </property>
   <widget class="QStackedWidget" name="stackLR">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>811</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_log">
     <widget class="QPushButton" name="btn_log">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>270</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_log{
	color: rgb(255, 255, 255);
	background-color: rgb(26, 115, 232);
	border:2px solid rgb(26, 115, 232);
	border-radius:8px;
}
#btn_log:hover{
	color: rgb(26, 115, 232);
	background-color: rgb(255, 255, 255);
}
#btn_log:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
      </property>
      <property name="text">
       <string>登    录</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_user_log">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>291</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
border-color: rgb(2, 198, 211);
</string>
      </property>
      <property name="placeholderText">
       <string>账号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_pwd_user">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>291</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
border-color: rgb(2, 198, 211);
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_reg">
     <widget class="QPushButton" name="btn_reg">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>290</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_reg{
	color: rgb(255, 255, 255);
	background-color: rgb(26, 115, 232);
	border:2px solid rgb(26, 115, 232);
	border-radius:8px;
}
#btn_reg:hover{
	color: rgb(26, 115, 232);
	background-color: rgb(255, 255, 255);
}
#btn_reg:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
      </property>
      <property name="text">
       <string>注    册</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_pwd1">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>291</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(2, 198, 211);
border:1px solid rgb(0,0,0);
border-radius:8px;
border-color: rgb(2, 198, 211);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_pwd2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>291</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
border-color: rgb(2, 198, 211);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>确认密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_name_reg">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>291</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
border-color: rgb(2, 198, 211);
</string>
      </property>
      <property name="placeholderText">
       <string>名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_reg">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>291</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
border-color: rgb(2, 198, 211);
</string>
      </property>
      <property name="placeholderText">
       <string>账号：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_set_2"/>
   </widget>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>0</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/del.png</normaloff>:/res/del.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_log_sw">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>600</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>登录</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/login.png</normaloff>:/res/login.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reg_sw">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>600</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>注册</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/reg.png</normaloff>:/res/reg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1081</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(://res/bg.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_bg</zorder>
   <zorder>stackLR</zorder>
   <zorder>btn_quit</zorder>
   <zorder>btn_log_sw</zorder>
   <zorder>btn_reg_sw</zorder>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
